src/tls/context.h

```diff
@@ -17,15 +17,8 @@ namespace tls
     EntropyPtr entropy;
 
 #ifndef NO_STRICT_TLS_CIPHERSUITES
-#  ifdef MOD_MBEDTLS
```

Comment:
Could you also remove the dead value in the `ParticipantsCurve` enum in `network.py`?

Reply:
Sure, will do!

Reply:
I'll go around the doc and scrub mentions of 25519 too while I'm at it, because we just don't support it at the moment.